packages/tools/pactjs-cli/src/contract-generate/generate.ts

```diff
@@ -1,13 +1,14 @@
 import {
   FileContractDefinition,
   generateDts,
-  IContractDefinition,
+  generateDts2,
+  pactParser,
   StringContractDefinition,
 } from '@kadena/pactjs-generator';
 
 import { retrieveContractFromChain } from '../utils/retrieveContractFromChain';
 
-import { ContractGenerateOptions } from './';
+import { IContractGenerateOptions } from './';
```

Comment:
Can we import from the file instead of the barrel file? Otherwise we inadvertently import everything from the barrel file